internal/scheduleringester/instructions.go

```diff
@@ -216,7 +248,14 @@ func (c *InstructionConverter) handleJobRunErrors(jobRunErrors *armadaevents.Job
 				JobID: jobId,
 				Error: bytes,
 			}
-			markRunsFailed[runId] = &JobRunFailed{LeaseReturned: runError.GetPodLeaseReturned() != nil}
+			runAttempted := false
```

Comment:
should this default to true?

Reply:
Probably yes

It is a bit awkward because the value only matters if lease was returned - so its value is somewhat irrelevant for other cases

Reply:
In practice it probably doesn't matter too much- although I'd argue that in general it's safer to assume the run was attempted if we can't determine whether it was.